.pipelines/rp-int-deploy.yml

```diff
@@ -0,0 +1,88 @@
+trigger: none
+pr: none
+
+parameters:
+  - template: rp-params.yml
+
+resources:
+  containers:
+    - container: container
+      image: registry.access.redhat.com/ubi8/toolbox:8.8
+      options: --user=0 --privileged -v /dev/shm:/dev/shm --device /dev/net/tun --name vpn
+
+variables:
+  # - template: vars.yml # probably redundant
+  - template: rp-vars.yml
+
+jobs:
+  - job: RP_int_deployment
+    timeoutInMinutes: 180
+    pool:
+      name: 1es-aro-ci-pool
+
+    steps:
+# setup
+      - template: ./templates/template-checkout.yml
+      - template: ./templates/template-az-cli-login.yml # does it have az CLI alraedy?
```

Comment:
It should already have az, hopefully. I checked over our pipelines (such as e2e) and we use `az` there, and it doesn't appear to be explicitly installed by the pipeline.

Reply:
I assume it comes from `registry.access.redhat.com/ubi8/toolbox:8.8`. Yesterday @tony-schndr @rolandmkunkel were wondering about it as well.